Formula/todolist.rb

```diff
@@ -0,0 +1,51 @@
+require "language/go"
+
+class Todolist < Formula
+  desc "Very fast, simple task manager for the command-line, based upon GTD."
+  homepage "http://todolist.site"
+  url "https://github.com/gammons/todolist/archive/0.2.0.tar.gz"
+  sha256 "13882bbbbcb86e05d6729f5bab2507de3aeabc1b4b4b34008480ee7549212c86"
+
+  head "https://github.com/gammons/todolist.git"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/mattn/go-isatty" do
+    url "https://github.com/mattn/go-isatty.git", :revision => "56b76bdf51f7708750eac80fa38b952bb9f32639"
+  end
+
+  go_resource "github.com/mattn/go-colorable" do
+    url "https://github.com/mattn/go-colorable", :revision => "9056b7a9f2d1f2d96498d6d146acd1f9d5ed3d59"
```

Comment:
Sorry, I think I was unclear before 😢. I meant in all cases where `:using => :git` was specified for GitHub stuff it was safe to simply append the `.git` to the URL. Needs to be done for every `go_resource` here rather than just one.